### DISCUSSION TITLE
New movement .

### DISCUSSION THREAD

**Discussion by SALAUDDIN06** [community/community#110251] [GitHub Education] [answered]
### Hi everyone,

share your first moment to be a part of community  what  new things you have learnt for the first time after joining the comminity?

**Comment by SALAUDDIN06** [comment]
New programs and creating new repositories commits made for the first time.

**Comment by naveenalla3000** [comment, marked as answer]
Collaboration and pair programming with other programmers around the world and version tracking of code and various branching stages with Git.
